jakarta-eclipse/src/org/jakarta/codeAction/CodeActionHandler.java

```diff
@@ -0,0 +1,69 @@
+package org.jakarta.codeAction;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.core.ICompilationUnit;
+import org.eclipse.jdt.core.ISourceRange;
+import org.eclipse.jdt.core.dom.CompilationUnit;
+import org.eclipse.jdt.core.manipulation.CoreASTProvider;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.CodeActionKind;
+import org.eclipse.lsp4j.Diagnostic;
+import org.jakarta.jdt.DiagnosticsCollector;
+
+import io.microshed.jakartals.commons.JakartaJavaCodeActionParams;
+
+import org.jakarta.jdt.JDTUtils;
+import org.jakarta.jdt.DiagnosticsCollector;
+import org.jakarta.codeAction.HttpServletQuickFix;
+
+public class CodeActionHandler{
```

Comment:
Let's credit where we based this code off of, with the original file header

Reply:
We should add a javadoc comment on this class with a link to where this class came from/was modified from, same applies to the other classes